docs/victoriametrics-cloud/deployments/access-tokens.md

```diff
@@ -0,0 +1,179 @@
+---
+title : "How to write data"
+weight: 2
+menu:
+  docs:
+    parent: "deployments"
+    weight: 2
+---
+
+You need to use Access tokens to access the deployment.
```

Comment:
@jgomezselles could you please verify the text, i guess we need to start explaining what is it first and after that why people need it